### PR TITLE
Update services for protocol 6.

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Update tools for protocol version 6 and node version 6.

Addresses #75 

## Checklist

- [ ] My code follows the style of this project.
- [ ] The code compiles without warnings.
- [ ] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Review by EmilLa1 — APPROVED** [review]

File: state-compare/README.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ The following checks are performed.
   tool checks that the pools are the same, and have the same capital. It also
   checks that the delegators for each pool are the same, which includes checking
   that the staked amounts are the same, as well as any pending change.
+- Update sequence numbers are migrated correctly.
```
Comment (EmilLa1) [inline state-compare/README.md:48]:
```suggestion
- Update sequence numbers are migrated correctly.
- Election difficulty only checked for protocols before P6.
```

File: state-compare/src/main.rs [hunk 2]
```diff
@@ -534,14 +572,19 @@ async fn compare_active_bakers(
...
+    pv2: ProtocolVersion,
 ) -> anyhow::Result<bool> {
     println!("Checking active bakers.");
     let mut found_diff = false;
     let (ei1, ei2) = futures::try_join!(
         client1.get_election_info(block1),
         client2.get_election_info(block2)
     )?;
-    if ei1.response.election_difficulty != ei2.response.election_difficulty {
+    if pv1 < ProtocolVersion::P6
```
Comment (EmilLa1) [inline state-compare/src/main.rs:584]:
```suggestion
    if pv2 < ProtocolVersion::P6
```

Comment (abizjak) [reply state-compare/src/main.rs:584]: I am checking that both pv1 and pv2 are below P6. I suppose I could just check pv2 and not both.

Or am I missing something?

Comment (abizjak) [reply state-compare/src/main.rs:584]: Discussed offline, will keep it as is.

**Pull request merged**